sdk/common/src/main/scala/org/typelevel/otel4s/sdk/Attributes.scala

```diff
@@ -73,17 +63,43 @@ final class Attributes private (
     Show[Attributes].show(this)
 }
 
-object Attributes {
+object Attributes extends SpecificIterableFactory[Attribute[_], Attributes] {
+  private val Empty = new MapAttributes(Map.empty)
 
-  val Empty = new Attributes(Map.empty)
+  /** Creates [[Attributes]] with the given `attributes`.
+    *
+    * @note
+    *   if there are duplicated keys in the given `attributes`, only the last
+    *   occurrence will be retained.
+    *
+    * @param attributes
+    *   the attributes to use
+    */
+  override def apply(attributes: Attribute[_]*): Attributes =
+    fromSpecific(attributes)
```

Comment:
I don't think you actually need this override—I think it already delegates to `fromSpecific`. unless you're doing it for the docs, in which case calling `super.apply` instead might make that clearer? idk. don't care hugely

Reply:
I do it only for the docs. `super.apply(attributes: _*)` requires to turn arguments into varargs :(